internal/repository/transaction_repo.go

```diff
@@ -50,3 +52,18 @@ func (r *transactionRepoImpl) FindOne(_ context.Context, id string) (models.Tran
 
 	return transaction, nil
 }
+
+func (r *transactionRepoImpl) Create(_ context.Context, transaction models.Transaction) error {
+	if transaction.Sender == "" || transaction.Receiver == "" || transaction.Owner == "" {
```

Comment:
You could return something like 
```go
if transaction.Sender == "" {
   return errors.Wrap(ErrMissingFields, "sender")
}

if transaction.Receiver == "" {
   return errors.Wrap(ErrMissingFields, "receiver")
}

if transaction.Owner == "" {
   return errors.Wrap(ErrMissingFields, "owner")
}
```

Reply:
There are pros and cons, but it is more interesting to be more specific about the errors to make it easier to debug
Another possibility is to create specific errors from another

```go
var (
      ErrMissingField  = errors.New("transaction field is missing")
      ErrMissingSenderField = errors.Wrap(ErrMissingField, "sender")
      ErrMissingReceiverField = errors.Wrap(ErrMissingField, "receiver")
      ErrMissingOwnerField = errors.Wrap(ErrMissingField, "owner")
)
```

By doing this, you can be memory efficient by re-using the same error pointer throughout the lifetime of your application.

The other benefit is handling the error and changing how your application reacts. 

For example
```go
if errors.Is(err, ErrMissingField) {
  log.Error().Warn(err).Msg(err.Error())
  utils.ErrorWithMessage(w, http.StatusBadRequest, err.Error())
}
```